src/modules/configurator/components/discount/Discount.tsx

```diff
@@ -0,0 +1,16 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import styles from './Discount.styles';
+
+export const Discount: React.FC = () => {
+  return (
+    <div css={styles.discount__container}>
```

Comment:
Prouci malo jos BEM naming css klasa http://getbem.com/introduction/ ( ako si to pokusao koristiti ).
Ovdje imas blok  _discount_
Zatim imas elemente: input i button - znaci bilo bi: _discount__input_ i _discount__button_
U slucaju da imas jos neki modifier npr. drugu verziju buttona bilo bi npr. _discount__button--secondary_